docs/_data/nakadi-event-bus-api.yaml

```diff
@@ -964,6 +964,11 @@ paths:
           required: false
           default: 0
           minimum: 0
+        - name: show_status
```

Comment:
do you think it is better to have separate endpoint for that ?

Reply:
@adyach I followed the requirements in our internal ticket:
`GET /subscriptions endpoint should have additional parameter show_status = true/false (false by default).`

TO be honest I don't really have a preference one way or another. Do you think it would be better as a separate endpoint?